website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step01-config.sh

```diff
@@ -1,4 +1,4 @@
-export KARPENTER_VERSION={{< param "latest_release_version" >}}
+export KARPENTER_VERSION=v0.23.0
```

Comment:
Does this not work? Needs to dynamically update as we release versions.

Reply:
I don't know where that came from, but it wasn't working in the netlify preview so I just added the literal v0.23.0 value.  All previous docs releases also have the literal values for the current release.

Reply:
I fixed it. The `latest_release_version` is working now.